.vibedev/specs/hugegraph-connector/design.md

```
@@ -0,0 +1,649 @@
+    Event --> Sink
+    Sink --> Factory
+    Factory --> SinkImpl
+    SinkImpl --> Writer
+    Writer --> Mapper
+    Writer --> Client
+    Client --> GraphDB
+```
+
+### 模块分层
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)